services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -82,6 +82,11 @@ class ComputationalBackendSettings(BaseCustomSettings):
         FileLinkType.PRESIGNED,
         description=f"Default file link type to use with computational backend on-demand clusters '{list(FileLinkType)}'",
     )
+    COMPUTATIONAL_BACKEND_MAX_WAITING_FOR_CLUSTER_TIMEOUT: datetime.timedelta = Field(
```

Comment:
Use `Annotated`. You can try the dedicated prompt here if you like

Reply:
so for info the prompt had to run twice to do what it was supposed to do. I hope the test will catch whatever it might have done wrong.